protected/tests/functional_test/AdminSiteAccessTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Functional test for only logged in admin user can access /site/admin.
+ */
+
+class AdminSiteAccessTest extends FunctionalTesting
+{
+    use BrowserSignInSteps;
+    use BrowserPageSteps;
+
+    public function testItShouldBeDisplayedToUsersWithAdminRole ()
+    {
+        // Logged in as Admin, and can visit to /site/admin page.
+        // 2 assertions, 2 passes.
+        $this->loginToWebSiteWithSessionAndCredentialsThenAssert("admin@gigadb.org","gigadb","Admin");
```

Comment:
Instead of `Admin`, I think it is more specific to check for `Joe's GigaDB Page` after the user has logged in.